src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -786,6 +786,7 @@ export class BrightScriptDebugSession extends BaseDebugSession {
                         }
                         response.body = {
                             result: v.value,
+                            type: v.type,
```

Comment:
We weren't passing the variable type back to the debug protocol, so this is what enables the type over in the watch/variables panel